muspinsim_input/muspinsim_input.xml

```diff
@@ -180,72 +192,87 @@
                     </repeat>
                 </when>
             </conditional>
-            <param name="average_axes" type="select" display="checkboxes" multiple="true" optional="true" value="orientation" label="Choose keywords that should have an average carried out over them"
-                help="Any keywords not included here or as x-axis are included in separate files">
+            <param name="average_axes" type="select" display="checkboxes" multiple="true" optional="true" value="orientation" label="Average axes"
+                   help="keywords that should have an average carried out over them. Each keyword ticked should have a range specified.
+                   Keywords not ticked or set as X axis with a range will be included in separate files">
                 <option selected="true" value="orientation">orientation</option>
                 <option value="polarization">polarization</option>
+                <option value="field">field</option>
+                <option value="time">time</option>
+                <option value="temperature">temperature</option>
             </param>
-            <param name="euler_convention" type="select" display="radio" label="Define euler angle convention (ignored if euler angles not defined)">
-                <option selected="true" value="zyz">zyz</option>
-                <option value="zxz">zxz</option>
-            </param>
-            <repeat name="orientations" title="Orientations to use for crystallites - (define powder averages)">
+            <repeat name="orientations" title="Orientations" help="Orientations to use for crystallites - (define powder averages)">
                 <conditional name="orientation_options">
-                    <param name="orientation_preset" type="select" display="radio" label="Choose orientation type to define" optional="false"
-                           help="Polar angles: Define two polar angles theta and phi, defining only the direction of the z-axis (recommended for powder averages).
-                                Euler angles: Define 3 euler angles defining a new frame, convention used is ZYZ by default.
-                                Euler angles with weight: define 3 euler angles and a weight (will be normalized automatically).
+                    <param name="orientation_preset" type="select" display="radio" label="Orientation" optional="false"
+                           help="Polar angles: Define two polar angles θ and ϕ, defining only the direction of the z-axis (recommended for powder averages).
```

Comment:
This hint text is clear and well-formatted - even though it's long, it's pretty easy to read. This is a good example to follow for formatting.